src/domain/event/eventCard/__tests__/LargeEventCard.test.tsx

```diff
@@ -102,7 +102,7 @@ test('should go to event page by click Read more button', () => {
     })
   );
 
-  expect(history.location.pathname).toEqual('/fi/event/123');
```

Comment:
if backward compability was left, we could check both `event` and `events`?

Reply:
yeah maybe, but that is a totally different test in a different place?

Reply:
that compability has nothing to do with LargeEventCard functionality